tests/unit/algorithms/classification/adapters/openvino/test_openvino_models.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import numpy as np
+
+from openvino.model_zoo.model_api.adapters import create_core
+from otx.algorithms.classification.tasks.openvino import OpenvinoAdapter
+from openvino.model_zoo.model_api.models.classification import Classification
+from otx.algorithms.classification.adapters.openvino.model_wrappers import (
+    OTXClassification,
+)
+from tests.test_suite.e2e_test_system import e2e_pytest_unit
+from tests.unit.api.parameters_validation.validation_helper import (
+    check_value_error_exception_raised,
+)
+from tests.unit.algorithms.classification.test_helper import (
+    generate_label_schema,
+    generate_cls_dataset
+)
+from otx.algorithms.classification.utils.labels_utils import get_multihead_class_info
+
+
+class MockClassification(OTXClassification, Classification):
+    def __init__(self):
+        self.out_layer_name = 'logits'
+        hierarchical_dataset = generate_cls_dataset(hierarchical=True)
+        label_schema = generate_label_schema(hierarchical_dataset.get_labels(),
+                                         multilabel=False,
+                                         hierarchical=True)
+        self.multihead_class_info = get_multihead_class_info(label_schema)
+
+
+class TestOTXClassification:
+    @pytest.fixture(autouse=True)
+    def setup(self, mocker) -> None:
+        self.cls_ov_model = MockClassification()
+        self.fake_logits = np.array([-0.12, 0.92])
+        self.fake_logits_hierarchical = np.array([3.01, -2.75, -0.63, 0.74, 0.017, -0.12, -0.42, -0.88])
+
+    @pytest.mark.parametrize("multilabel, hierarchical", [(False, False), (True, False), (False, True)])
+    @e2e_pytest_unit
+    def test_postprocess(self, multilabel, hierarchical, mocker):
+        self.cls_ov_model.multilabel = multilabel
+        self.cls_ov_model.hierarchical = hierarchical
+        fake_model_out = {'logits': self.fake_logits}
+        if self.cls_ov_model.hierarchical:
+            fake_model_out = {'logits': self.fake_logits_hierarchical}
+
+        fake_metadata = {'original_shape': (254, 320, 3), 'resized_shape': (224, 224, 3)}
+        prediction = self.cls_ov_model.postprocess(fake_model_out, fake_metadata)
+        if self.cls_ov_model.hierarchical:
+            assert len(prediction) == 3
+        else:
+            assert prediction[0][0] == 1
+            assert len(prediction) == 1
+
+    @pytest.mark.parametrize("multilabel, hierarchical", [(False, False), (True, False), (False, True)])
+    @e2e_pytest_unit
+    def test_aux_postprocess(self, multilabel, hierarchical, mocker):
```

Comment:
Could you make the test function easy to be understood that which function is tested?
Maybe you can add comments or change test function name similar to target function.
It would be better to update other functions also.